plugins/system_upgrade.py

```diff
@@ -262,7 +262,12 @@ def pick_boot(message_id, n):
 
 def show_log(n):
     boot_id = pick_boot(ID_TO_IDENTIFY_BOOTS, n)
-    check_call(['journalctl', '--boot', boot_id.hex])
+    process = Popen(['journalctl', '--boot', boot_id.hex])
+    process.wait()
+    rc = process.returncode
+    if rc == 1:
```

Comment:
Not in this case, rc 13 is returned when you close the output. According to my tests, only 1 is returned in cases of incorrect entry, and incorrect format of entry. I would prefer to use here less aggressive error detection.